### PR TITLE
Add "CloudWatch Metrics Disabled" query for CloudFormation #2544

### LINKED ISSUE #2544: Add "CloudWatch Metrics Disabled" query for CloudFormation
### Platform
CloudFormation

### Provider
AWS

### Description
See if CloudWatch Metrics is Enabled

### PR DESCRIPTION
**Pull request by pedro-mimoso**
Closes #2544 

**Proposed Changes**

- Add "CloudWatch Metrics Disabled" query for CloudFormation;
- Change Terraform "CloudWatch Metrics Not Enabled" query name to "CloudWatch Metrics Disabled";

I submit this contribution under Apache-2.0 license.

### CODE REVIEW

**Review by Ruben-Silva — APPROVED** [review]
Looks good to me!

File: assets/queries/terraform/aws/cloudwatch_metrics_disabled/metadata.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "id": "081069cb-588b-4ce1-884c-2a1ce3029fe5",
-  "queryName": "CloudWatch Metrics Not Enabled",
+  "queryName": "CloudWatch Metrics Disabled",
   "severity": "MEDIUM",
   "category": "Observability",
-  "descriptionText": "Enable the CloudWatch Metrics",
+  "descriptionText": "See if CloudWatch Metrics is Enabled",
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/aws/cloudwatch_metrics_disabled/metadata.json:6]: `"Checks if..."`

File: assets/queries/cloudFormation/cloudwatch_metrics_disabled/metadata.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "5d3c1807-acb3-4bb0-be4e-0440230feeaf",
+  "queryName": "CloudWatch Metrics Disabled",
+  "severity": "MEDIUM",
+  "category": "Observability",
+  "descriptionText": "See if CloudWatch Metrics is Enabled",
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/cloudwatch_metrics_disabled/metadata.json:6]: `"Checks if..."`

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]
let's improve the description

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Pull request merged**